packages/signature-v4/src/SignatureV4.ts

```diff
@@ -139,6 +138,7 @@ export class SignatureV4
 
     const {
       signingDate = new Date(),
+      expiration = new Date(Date.now() + 3600 * 1000),
```

Comment:
Where does this 1 hr expiration come from?

Reply:
It comes from v2 https://github.com/aws/aws-sdk-js/blob/46b36d10f7397b33be90c86642b2fea1358eb681/lib/signers/presign.js#L91